This version always assumes the kenrel is newer than 2.4.0.
But it is compatible with older compilers that dont allow
__attribude__((constructor(value))) (e.g. gcc 4.2)

diff --git a/proc/sysinfo.c b/proc/sysinfo.c
index fc05076..386faad 100644
--- a/proc/sysinfo.c
+++ b/proc/sysinfo.c
@@ -222,11 +222,11 @@ static void init_libproc(void){
   smp_num_cpus = sysconf(_SC_NPROCESSORS_ONLN);
   if(smp_num_cpus<1) smp_num_cpus=1; /* SPARC glibc is buggy */
 
-  if(linux_version_code > LINUX_VERSION(2, 4, 0)){ 
+//  if(linux_version_code > LINUX_VERSION(2, 4, 0)){ 
     Hertz = find_elf_note(AT_CLKTCK);
     if(Hertz!=NOTE_NOT_FOUND) return;
     fputs("2.4+ kernel w/o ELF notes? -- report this\n", stderr);
-  }
+//  }
   old_Hertz_hack();
 }
 
